static/sass/_pattern_cube-progression.scss

```diff
@@ -0,0 +1,201 @@
+    }
+
+    &.is-commands {
+      @extend %diamond-badge;
+
+      background-image: url(https://assets.ubuntu.com/v1/5e068487-Commands.svg);
+      transform: translate(map-get($diamondSize, "width") * 2px, 0);
+    }
+
+    &.is-security {
+      @extend %rhombus-badge;
+
+      background-image: url(https://assets.ubuntu.com/v1/d9074c81-Security.svg);
+      transform: translate(
+        map-get($diamondSize, "width") * 2.5px,
+        map-get($diamondSize, "height") * 0.5px
+      );
+    }
+
+    &.is-networking {
+      @extend %rhombus-badge;
+
+      background-image: url(https://assets.ubuntu.com/v1/3f3f2687-Networking.svg);
+      transform: translate(
+        map-get($diamondSize, "width") * 2px,
+        map-get($diamondSize, "height") * 0.5px
+      );
+    }
+
+    &.is-kernel {
+      @extend %diamond-badge;
+
+      background-image: url(https://assets.ubuntu.com/v1/7b4466fe-Kernel.svg);
+      transform: translate(
+        map-get($diamondSize, "width") * 0.5px,
+        map-get($rhombusSize, "height") * 1px
+      );
+    }
+
+    &.is-microk8s {
+      @extend %rhombus-badge;
+
+      background-image: url(https://assets.ubuntu.com/v1/c26080a5-Microk8s.svg);
+      transform: translate(
+        map-get($diamondSize, "width") * 1px,
+        (map-get($rhombusSize, "height") * 1px) +
+          (map-get($diamondSize, "height") * 0.5px)
+      );
+    }
+
+    &.is-virtualisation {
+      @extend %rhombus-badge;
+
+      background-image: url(https://assets.ubuntu.com/v1/4f1c55a5-Virtualisation.svg);
+      transform: translate(
+        map-get($diamondSize, "width") * 0.5px,
+        (map-get($rhombusSize, "height") * 1px) +
+          (map-get($diamondSize, "height") * 0.5px)
+      );
+    }
+
+    &.is-storage {
+      @extend %diamond-badge;
+
+      background-image: url(https://assets.ubuntu.com/v1/090ef9e2-Storage.svg);
+      transform: translate(
+        map-get($diamondSize, "width") * 1.5px,
+        map-get($rhombusSize, "height") * 1px
+      );
+    }
+
+    &.is-juju {
+      @extend %rhombus-badge;
+
+      background-image: url(https://assets.ubuntu.com/v1/ce96b82f-Juju.svg);
+      transform: translate(
+        map-get($diamondSize, "width") * 2px,
+        (map-get($rhombusSize, "height") * 1px) +
+          (map-get($diamondSize, "height") * 0.5px)
+      );
+    }
+
+    &.is-maas {
+      @extend %rhombus-badge;
+
+      background: url(https://assets.ubuntu.com/v1/711b3e71-MAAS.svg);
+      transform: translate(
+        map-get($diamondSize, "width") * 1.5px,
+        (map-get($rhombusSize, "height") * 1px) +
+          (map-get($diamondSize, "height") * 0.5px)
+      );
+    }
+
+    &.is-tag {
+      @extend %diamond-badge;
+
+      background-image: $color-accent;
+      clip-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
+      transform: translate(
+        map-get($diamondSize, "width") * -0.5px,
+        map-get($rhombusSize, "height") * 1px
+      );
+    }
+
+    .is-faded {
```

Comment:
Can you make the `.is-faded` styling a placeholder which is extended in multiple places instead of extending a class please.